kmir/src/tests/integration/data/prove-rs/slice-split-at.rs

```diff
@@ -0,0 +1,7 @@
+fn main() {
+    let arr: [u8; 2] = [42, 255];
+    let slice = arr.as_slice();
+    let (left, right) = slice.split_at(1);
+    assert!(left == [42]);
+    assert!(right == [255]);
```

Comment:
The `Context` changes are not quite right but this test does not show this because no values are written through a reference with offset.